moped-editor/src/views/staff/NewStaffView.js

```diff
@@ -40,7 +40,12 @@ const validationSchema = yup.object().shape({
   first_name: yup.string().required(),
   last_name: yup.string().required(),
   title: yup.string().required(),
-  workgroup_id: yup.number().integer().required(),
+  workgroup_id: yup
+    .number()
+    .integer()
+    .required()
+    // see https://github.com/jquense/yup/issues/211
+    .typeError("workgroup is a required field"),
```

Comment:
The Yup error message that comes through here by default is not user friendly (complains about `NaN` and not about the required part) so I set a message that matches up with the other fields.